delivery_package_default_shipping_weight/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+{
+    "name": "Delivery Package Default Shipping Weight",
+    "summary": "Set default package shipping weight according to packaging",
+    "version": "14.0.1.0.0",
+    "development_status": "Beta",
```

Comment:
Shouldn't this changed ?

Reply:
Hi, should we go for Production/Stable already or wait a few weeks after it's been merged?

Reply:
we are using it since more than 1 year on prod... should be ok to switch to stable

Reply:
good day @simahawk , I pushed the update for `development_status`